packages/backend/src/assets/ai.json

```diff
@@ -303,6 +303,19 @@
       },
       "sha256": "893ae2442b36b2e8e1134ccbf8c0d9bd670648d0964509202ab30c9cbb3d2114",
       "backend": "none"
+    },
+    {
+      "id": "hf.MaziyarPanahi.Mistral-7B-Instruct-v0.3.Q4_K_M",
```

Comment:
should it come from https://huggingface.co/mistralai rather than "hf.MaziyarPanahi"

Reply:
They don't provide GGUF files thus this repo